docs/project.md

```diff
@@ -199,3 +202,16 @@ HEMTT will use the specified signature name as part of the full signature (`.bis
 ```
 
 Above will result in signature name of `TST_<addon>.pbo.my-1.0.0.0.bisign` (where `<addon>` is the name of the addon folder), located next to the matching addon PBO.
+
+
+## reuse_private_key
+
+**Type**: bool
+
+If set to `true`, HEMTT will use (and re-use) `releases/keys/{keyname}.biprivatekey`. It will be generated if it doesn't exist.
+
+The default behaviour is to generate a new private key each time and discard it immediately; HEMTT strongly recommends that you only re-use the key if you are making a client-side mod where it will not matter if clients are running different versions of the mod.
```

Comment:
docisfy has a nice helper for highlighting important info. https://docsify.js.org/#/helpers?id=important-content

It should be used here to highlight the recommendation.